seguro/common/broker.py

```diff
@@ -6,24 +6,47 @@
 from queue import Queue
 
 import paho.mqtt.client as mqtt
+from seguro.common.config import (
+    MQTT_HOST,
+    MQTT_PASSWORD,
+    MQTT_PORT,
+    MQTT_USERNAME,
+)
 
 
-class MQTTClient:
+class Broker:
```

Comment:
Why `Broker`? Isn't that a client, as the broker is run as another service?

Reply:
I changed it to `BrokerClient`